twa-matching.md

```diff
@@ -0,0 +1,24 @@
+# Project TWA Careers and Candidates Matching
+
+## Client Profile
+
+The Transformative Workforce Academy (TWA) is an initiative at Saint Louis University aimed at supporting individuals at risk of incarceration to lead productive lives without returning to prison. SLU's comprehensive approach includes prevention, in-prison, and reentry services. TWA specifically focuses on helping individuals leaving prison to find and maintain meaningful employment.
+
+TWA started its efforts in 2018 by organizing the Second Chance Job Fair in partnership with federal and state probation and parole offices. The fair aimed to connect job seekers with employers who consider candidates holistically, beyond their criminal records. Building on the success of these job fairs, TWA now aims to provide year-round support to jobseekers and employers through the Second Chance Job Portal. The Second Chance Job Portal will offer resources and assistance for jobseekers seeking second-chance employment, as well as support for employers interested in hiring and retaining this often overlooked segment of the workforce. The goal is to facilitate successful employment opportunities and help address the root causes of crime by providing ongoing support and resources to those seeking a second chance.
+
+## Project Brief
+
+This project involves collecting and matching the below information
+
+1. criminal records of individuals
+2. Job requirements from the hiring team.
+
+If the above 1 and 2 requirements are not related to each other. then we can send this particular profile to hiring team. similarly vice versa.
+
+## Functional Requirements
+
+* Profile building.
+* Job posting.
+* Apply for jobs.
+* Filtering candidates.
+* Training candidates.
```

Comment:
Reporting and Analytics we can add this as functional requirement where we can provide reports and analytics on the recruitment process. which provides an insight how many applications are received, how many got accepted, and other relevant metrics.

Reply:
incorporated comments into updates.